template/README.template.md

```diff
@@ -102,6 +102,31 @@ For more information on Ansible collection usage, see [Ansible's official usage
 
 Use-case examples for this collection can be found [here](./examples/README.md).
 
+## Development
```

Comment:
Unless anyone has any objections, I plan on making a ticket to flesh this section out when I have time 🙂